lib/evm_database_ops.rb

```diff
@@ -29,13 +29,13 @@ def self.database_size(opts)
     PostgresAdmin.database_size(opts)
   end
 
-  def self.validate_free_space(database_opts)
-    free_space = backup_destination_free_space(database_opts[:local_file])
+  def self.validate_free_space(target_uri, dbname)
+    free_space = backup_destination_free_space(target_uri)
     db_size = database_size(database_opts)
```

Comment:
Looks like you're still looking for `database_opts` here, but it's been removed.

Reply:
Ah butts... maybe I can't make this to the method args.

Oh well, will fix.  Thanks!